exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,7 @@
 // Declare your function here
-
+function createLongGreeting(name1, age) {
+    return "Hello, my name is " + name1 + " and I'm " + age + " years old.";  
```

Comment:
Hi Omar,
Your code looks good. For a challenge on this one you could try using the template literal syntax instead. 
(Link here if needed: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
Thanks for your input @nPelvin . I have made the changes.